src/main/knn/KnnGraphTester.java

```diff
@@ -381,6 +404,16 @@ private void run(String... args) throws Exception {
       indexSizeOnDiskMB = indexSizeOnDiskBytes / 1024. / 1024.;
       System.out.println(String.format(Locale.ROOT, "index disk uage is %.2f MB", indexSizeOnDiskMB));
     }
+    try (Directory dir = FSDirectory.open(indexPath); IndexReader reader = DirectoryReader.open(dir)) {
```

Comment:
Hmm this new code chunk is dup of chunk above?  Prolly accidental on conflict resolution...?

Reply:
Whoa! Thanks Mike, didn't realize when that snuck in. Fixed it now.